code/game/area/areas/mining.dm

```diff
@@ -287,3 +287,10 @@
 /area/icemoon/underground/explored // ruins can't spawn here
 	name = "Icemoon Underground"
 	area_flags = UNIQUE_AREA
+
+/area/icemoon/underground/explored/graveyard
+	name = "Graveyard"
+	area_flags = UNIQUE_AREA
+	ambience_index = AMBIENCE_SPOOKY
+	icon = 'icons/area/areas_station.dmi'
+	icon_state = "graveyard"
```

Comment:
So does the chapel's graveyard also have this area?

Reply:
Not presently. I'm going to add a Chapel Graveyard subtype of the area and put it there.